python/elastic_search_serialization.py

```diff
@@ -0,0 +1,514 @@
+import json
+import elasticsearch
+import mgp
+from typing import Any, List, Dict, Tuple
+from elasticsearch import Elasticsearch
+from elasticsearch.helpers import streaming_bulk
+from datetime import datetime
+
+# Elasticsearch constants
+ACTION = "action"
+INDEX = "index"
+ID = "_id"
+SOURCE = "source"
+SETTINGS = "settings"
+NUMBER_OF_SHARDS = "number_of_shards"
+NUMBER_OF_REPLICAS = "number_of_replicas"
+MAPPINGS = "mappings"
+DYNAMIC_TEMPLATES = "dynamic_templates"
+MAPPING = "mapping"
+ANALYZER = "analyzer"
+STRING = "string"
+EVENT_TYPE = "event_type"
+CREATED_VERTEX = "created_vertex"
+CREATED_EDGE = "created_edge"
+VERTEX = "vertex"
+EDGE = "edge"
+
+# Linkurious constants
+LK_TYPE = "lk_type"
+LKE_STRING = "_lke_string"
+LKE_NUMBER = "_lke_number"
+LKE_BOOLEAN = "_lke_boolean"
+LKE_DATE = "_lke_date"
+LK_CATEGORIES_HAS_RAW = "lk_categories_has_raw"
+LK_TYPE_HAS_RAW = "lk_type_has_raw"
+LK_CATEGORIES = "lk_categories"
+
+# Mappings of our data types
+lke_mapping: Dict[type, str] = {}
+lke_mapping[str] = LKE_STRING
+lke_mapping[int] = LKE_NUMBER
+lke_mapping[float] = LKE_NUMBER
+lke_mapping[bool] = LKE_BOOLEAN
+lke_mapping[datetime] = LKE_DATE
+
+# Create global logger object
+logger = mgp.Logger()
+
+# Create node index
+node_index_body: Dict[str, Any] = {
```

Comment:
Yes, the new version decouples this in a way that the user just specifies the path to the schema. Because also logically doesn't make sense to check if indexes exist every time he/she wants to index some document.